sap/rfc/user.py

```diff
@@ -62,6 +62,58 @@ def today_sap_date() -> str:
     return sap_date_from(datetime.date.today())
 
 
+class UserPasswordManager:
+    """An utility class for handling productive password"""
+
+    DUMMY_PASSWORD = 'DummyPwd123!'
```

Comment:
Would be nice if this can be configure.

Reply:
Sure. I can change it, but I would need more info about it. Should it be configured only for instance of the class? Or taken from some global storage, like environment variable? Or do you have already something similar, from which I can get the inspiration?